src/pages/o/[orgId]/events.tsx

```diff
@@ -5,13 +5,13 @@ import { QueryClient, useQuery } from 'react-query';
 function getEvents(orgId) {
     return async () => {
         try {
-            const cRes = await fetch(`http://api.zetk.in/v1/orgs/${orgId}/campaigns`);
+            const cRes = await fetch(`http://localhost:3000/api/orgs/${orgId}/campaigns`);
```

Comment:
The domain (as well as scheme/protocol and port) of the app should be externalized eventually, so that it can be run with other settings in production.